src/main.ts

```diff
@@ -1,15 +1,35 @@
 import Vue from 'vue';
 import AsyncComputed from 'vue-async-computed';
+import Notifications from 'vue-notification';
 
 import App from './App.vue';
 import vuetify from './plugins/vuetify';
 import router from './router';
 
 Vue.config.productionTip = false;
 Vue.use(AsyncComputed);
+Vue.use(Notifications);
 
 new Vue({
   vuetify,
   router,
   render: (h) => h(App),
 }).$mount('#app');
+
+declare global {
```

Comment:
@subdavis  Instead of adding function  directly to `window`, I would like to add then to an object called `slicerExtensions`  and be able to class `window.slicerExtensions.createNotice()`.

Do you have any suggestion on how to split and organize the code to achieve this ...

Since I am learning typescript, any pointers would be helpful.